framework/src/problems/FEProblemBase.C

```diff
@@ -5617,6 +5677,16 @@ FEProblemBase::init()
     es().init();
   }
 
+  // Now that the equation system and the dof distribution is done, we can generate the
+  // finite volume-related parts if needed.
+  if (haveFV())
+  {
+    _mesh.buildFiniteVolumeInfo();
+    _mesh.computeFiniteVolumeCoords();
+    _mesh.cacheFaceInfoVariableOwnership();
+    _mesh.cacheFVElementalDoFs();
```

Comment:
Would it make sense to have a single `MooseMesh` method that does all four of these calls?

Reply:
I can definitely do that! Just wanted to have the flexibility to call them all separately since some unit tests rely only a few of them.

Reply:
This reminds me that I'll need to add a unit test for elemental dof indices.